doc/cprover-manual/cbmc-tutorial.md

```diff
@@ -241,7 +241,7 @@ void unlock()
 int main()
 {
   unsigned got_lock = 0;
-  int times;
+  int times = 10;
```

Comment:
This makes the loop bounded, and thus the example not longer fits what the section is about.

Reply:
indeed, fixed